docs/kubernetes/README.md

```diff
@@ -69,6 +69,16 @@
    kubectl exec -it my-csi-app -- /bin/sh
    ```
 
+### Alternative Kubelet Directory
+
+Some Kubernetes distributions use a non-standard path for the Kubelet directory.
+The csi-driver needs to know about this to successfully mount volumes. You can
+configure this through the Helm Chart Value `node.kubeletDir`.
+
+- Standard: `/var/lib/kubelet`
+- **k0s**: `/var/lib/k0s/kubelet`
+- **microk8s**: `/var/snap/microk8s/common/var/lib/kubelet`
```

Comment:
Is microk8s always distributed through snap ? Do we have links to the upstream docs ?

Reply:
As far as I can tell microk8s is only distributed through Snap on Linux (its a Ubuntu product afterall): https://microk8s.io/docs/getting-started

Their official documentation for the NFS CSI driver also uses this path: https://microk8s.io/docs/how-to-nfs#install-the-csi-driver-for-nfs-2